driver/pparse.ml

```diff
@@ -166,6 +166,9 @@ let file ppf ~tool_name inputfile parse_fun ast_magic =
   close_in ic;
   apply_rewriters ~restore:false ~tool_name ast_magic ast
 
+let file ppf ~tool_name inputfile parse_fun ast_magic =
+  Timings.(time (Parsing inputfile))
+    (file ppf ~tool_name inputfile parse_fun) ast_magic
```

Comment:
I've had my own problems with `Pparse` when hacking on #351 . I think this module would gain from a redesign/refactor to cleanly expose the preprocessing step to the external world. This is for later, for now your choice is fine.